syncstorage-postgres/migrations/2025-10-20-155711_create_schema/up.sql

```diff
@@ -102,4 +87,4 @@ CREATE TABLE batch_bsos (
         collection_id,
         batch_id
     ) ON DELETE CASCADE
-);
+);
```

Comment:
Were these formatting changes from an automated tool?

Reply:
VS Code format on save.

Reply:
There's an included formatting with the official Postgres support for VSCode and also [PgFormatter](https://marketplace.visualstudio.com/items?itemName=bradymholt.pgformatter) which I think is the same one.